README.md

```diff
@@ -80,6 +80,7 @@ attribute value instead of the state value. `icon` lets you display an icon inst
 | toggle           | bool        | `false`                     | Display a toggle if supported by domain                            |
 | icon             | string/bool | `false`                     | Display default or custom icon instead of state or attribute value |
 | state_color      | bool        | `false`                     | Enable colored icon when entity is active                          |
+| default          | string      |                             | Display this value if the entity does not exist or should not be shown |
```

Comment:
* I am not sure what you mean with the first the question, as this is the exact description I have added. The description says "if the entity does not exist".
* "or should not be shown". There is a "hide_if" option. If the entity should not be shown because of this "hide_if" value/criteria matches, it would fall back to the default value.

Reply:
> * There is a "hide_if" option. If the entity should not be shown because of this "hide_if" value/criteria matches, it would fall back to the default value.

I suggest to explicitly mention the `hide_if` option in this description.

Reply:
Also, this is not only about `hide_if` option - seems that it works with `hide_unavailable` too

Reply:
https://github.com/benct/lovelace-multiple-entity-row/pull/303